lib/redix/stream/consumer.ex

```diff
@@ -111,6 +111,18 @@ defmodule Redix.Stream.Consumer do
     :ok
   end
 
+  def ack(consumer, id), do: GenServer.call(consumer, {:ack, id})
+
+  def handle_call({:ack, id}, _from,
+    state = %{
+      redix: redix,
+      stream: stream,
+      group_name: group_name
+    }) do
+    {:ok, num} = Redix.command(redix, ["XACK", stream, group_name, id])
```

Comment:
ah cool. this was all already there in the server state?

Reply:
Yea